Packs/EWS/Integrations/EWSv2/EWSv2.py

```diff
@@ -324,10 +324,7 @@
 
 PUBLIC_FOLDERS_ERROR = 'Please update your docker image to use public folders'
 if IS_PUBLIC_FOLDER and exchangelib.__version__ != "1.12.0":
-    if demisto.command() == 'test-module':
-        demisto.results(PUBLIC_FOLDERS_ERROR)
-        exit(3)
-    raise Exception(PUBLIC_FOLDERS_ERROR)
```

Comment:
why was this check if it is test module? maybe for better print in the config?

Reply:
I assume that we tried to avoid a traceback in the test button